docs/xy-plot.md

```diff
@@ -49,21 +49,22 @@ XYPlot is a wrapper for series, hints, axes and other components. Most of these
 * `opacity` (optional)
 * `fill` (optional)
 * `stroke` (optional)
+* `strokeWidth` (optional), `strokeStyle` (optional) - to control the width of the line series and whether they are dashed or solid. 
```

Comment:
@jckr, I suppose, you should put these changes separately: these properties don't support ranges, domains and types (as others do). I'd suggest to add them to `LineSeries`.